rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -632,20 +632,14 @@ static abstract class ObligationMixin extends Obligation {
                 "id",
                 "revision",
                 "type",
-                "title",
```

Comment:
These fields have been removed from blacklist. But I cannot see those in the response example you have shared. Is there anything I am missing here?